smsdk/client.py

```diff
@@ -583,6 +613,24 @@ def get_line_data(
             limit=limit, offset=offset, **kwargs
         )
 
+    xAxisTime = {
+        "unit": "",
+        "type": "datetime",
+        "data_type": "datetime",
+        "stream_types": [],
+        "raw_data_field": "",
+        "id": "endtime",
+        "title": "Time",
+        "isEnabled": True,
+    }
+    one_week_relative = {
+        "time_type": "relative",
+        "relative_start": 1,
+        "relative_unit": "week",
+        "ctime_tz": "America/Los_Angeles",
+    }
```

Comment:
What do these definition exist for?

Reply:
It might have crept in during code merging. It is not needed.